core/passport/github.js

```diff
@@ -2,54 +2,96 @@
  * core/passport/github.js
  * Setup Github Passport object
  *
- * @exports {Function} Server - Sets up Koa server with Github Passport
- * @exports {Object} Route - Koa router object for authentication
+ * @exports {Class} Strategy - Passport class for GitHub login
+ * @exports {Object} Route - Koa router object for GitHub authentication
  */
 
 import Passport from 'koa-passport'
 import Github from 'passport-github'
 import Router from 'koa-router'
 
-import { Config } from '~/app'
+import { Config, Log, Request } from '~/app'
 import { User } from '~/core/model/user'
 
+/**
+ * Updates user with rights given by GitHub
+ *
+ * @param {Object} user - user database object
+ * @return {Object} updated user object
+ */
+const getRights = async function (user) {
+  const promiseMember = function (member, username) {
+    let request = ''
+    if (typeof member === 'number') {
+      request = `https://api.github.com/teams/${member}/memberships/${username}`
+    } else {
+      request = `https://api.github.com/orgs/${member}/members/${username}`
+    }
+
+    return Request
+    .get(request)
+    .auth(user.github.access)
+    .then(data => {
```

Comment:
Nope. All chains have not been indented. Including `Request` and Promise chains like such:
```
    return cycle.release()
    .then(() => {}, ctx.throw('Unable to push release to repository', 500))
    .then(cycle.update({ _status: 'FINISH' }))
    .then(ctx.redirect('/dashboard'))
```